CODE_OF_CONDUCT.md

```diff
@@ -1,6 +1,6 @@
-# Communications Code of Conduct (CCOC)
+# Community Code of Conduct (CCOC)
```

Comment:
Typically open source projects adapts [code of conduct](https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md) from  [contributor covenant](https://www.contributor-covenant.org/). What's your thought on following a similar pattern and building on top of that. ​
​Also,  Naresh and I worked on baseline standards for one of the [repo](https://github.com/PublicisSapient/react-accelerate-modules) around contributing guidelines, RFCs, etc. and perhaps we can pick some work from there for this baseline repo.

Reply:
@cssmiles love the idea of reusing. And we definitely should look at everything we have for each standard we introduce at the org level.

The Contributor Convenant's code of conduct is one of two templates GitHub provides a template for. The other is a "Citizen's Code of Conduct" that is more for events, and not really appropriate for what we're doing here.

I like the one in this PR as it is derived from PS' core values. But I'm equally open to using Contributor Convenant as our starting point.

Reply:
I agree with you Anthony.  I like that it is based on our core values too. I think this is a great place to start.

Reply:
Are we able to add a default contribution markdown as CSP suggested?  I think that would be valuable to have as it's easy to forget to add those.